app/views/users/_profile.html.erb

```diff
@@ -1,144 +1,187 @@
-    <%= f.url_field :stackoverflow_url %>
-  </div>
-  <div class="field">
-    <%= f.label :linkedin_url, "LinkedIn URL" %>
-    <%= f.url_field :linkedin_url %>
-  </div>
-  <div class="field">
-    <%= f.label :behance_url, "Behance URL" %>
-    <%= f.url_field :behance_url %>
-  </div>
-  <div class="field">
-    <%= f.label :dribbble_url, "Dribbble URL" %>
-    <%= f.url_field :dribbble_url %>
-  </div>
-  <div class="field">
-    <%= f.label :medium_url, "Medium URL" %>
-    <%= f.url_field :medium_url %>
-  </div>
-  <div class="field">
-    <%= f.label :gitlab_url, "GitLab URL" %>
-    <%= f.url_field :gitlab_url %>
-  </div>
-  <div class="field">
-    <%= f.label :instagram_url, "Instagram URL" %>
-    <%= f.url_field :instagram_url %>
+</div>
+
+<%= form_for @user do |f| %>
+  <div class="crayons-card mb-6 grid grid-flow-row gap-6 p-6">
+    <div class="crayons-field">
+      <%= f.label :email, class: "crayons-field__label" %>
+      <%= f.text_field :email, class: "crayons-textfield", placeholder: "john.doe@example.com" %>
+    </div>
+
+    <div class="crayons-field crayons-field--checkbox">
+      <%= f.check_box :email_public, class: "crayons-checkbox" %>
+      <%= f.label :email_public, "Display email on profile", class: "crayons-field__label" %>
+    </div>
+
+    <div class="crayons-field">
+      <%= f.label :username, class: "crayons-field__label" %>
+      <%= f.text_field :username, maxlength: 30, class: "crayons-textfield", placeholder: "johndoe" %>
+    </div>
+
+    <div class="crayons-field">
+      <%= f.label :name, class: "crayons-field__label" %>
+      <%= f.text_field :name, maxlength: 100, class: "crayons-textfield", placeholder: "John Doe" %>
+    </div>
+
+    <div class="crayons-field">
+      <%= f.label :profile_image, class: "crayons-field__label" %>
+
+      <div class="flex items-center">
+        <% if @user.profile_image_url.present? %>
+          <span class="crayons-avatar crayons-avatar--xl mr-2"><img alt="<%= @user.username %> profile image" src="<%= ProfileImage.new(@user).get(width: 50) %>" class="crayons-avatar__image" /></span>
+          <%= f.file_field :profile_image, accept: "image/*", class: "crayons-card crayons-card--secondary p-3 flex items-center flex-1" %>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="crayons-field">
+      <%= f.label :website_url, class: "crayons-field__label" %>
+      <%= f.text_field :website_url, maxlength: 100, class: "crayons-textfield", placeholder: "https://yoursite.com" %>
+    </div>
+
+    <div class="crayons-field">
+      <%= f.label :summary, class: "crayons-field__label" %>
+      <%= f.text_area :summary, maxlength: 200, class: "crayons-textfield", placeholder: "A short bio...", rows: 3 %>
+    </div>
+
+    <div class="crayons-field">
+      <%= f.label :location, class: "crayons-field__label" %>
+      <%= f.text_field :location, maxlength: 100, class: "crayons-textfield", placeholder: "San Francisco, CA" %>
```

Comment:
Petition for the placeholder text to be "Halifax, Nova Scotia"

Reply:
Petition accepted!